contest/static/contest/js/contestArchive.js

```diff
@@ -42,6 +42,10 @@ $(document).ready(function() {
     });
 });
 
+function confirm_rejudge() {
+    return confirm("Are you sure you want to rejudge?");
```

Comment:
Can you ask user to enter total number of submissions need to rejudge? (Of course you have to tell the user first)
Just like when you want to delete a git repository, you need to enter your repository name.